src/Notes/ApplePayTBD.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * WooCommerce Admin TBD Apple Pay Note Provider.
+ *
+ * Adds a note to the merchant's inbox TBD something apple pay.
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+use \Automattic\WooCommerce\Admin\PluginsHelper;
+
+/**
+ * Apple_Pay_TBD
+ */
+class Apple_Pay_TBD {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-apple-pay-tbd';
+
+	/**
+	 * Get the note.
+	 *
+	 * @return Note|null
+	 */
+	public static function get_note() {
+		// Only show if Stripe plugin is active.
+		if ( ! PluginsHelper::is_plugin_active( 'woocommerce-gateway-stripe/woocommerce-gateway-stripe.php' ) ) {
+			return;
+		}
+
+		// Only show if in the US.
+		$base_location = wc_get_base_location();
```

Comment:
I did have some nice array destructuring here but key-based destructuring is only supported in PHP 7.2